opslevel/common.go

```diff
@@ -270,6 +270,22 @@ func flattenTagArray(tags []opslevel.Tag) []string {
 	return output
 }
 
+func flattenMembersArray(members []opslevel.UserId) []string {
+	output := []string{}
+	for _, member := range members {
+		output = append(output, member.Email)
```

Comment:
Same thing - the state should store IDs since they are immutable and aliases are not.  This creates determinisim for terraform.  If that messes up the diff's because a customer supplied an Email - then we just need to adjust our schema and code to handle that.

Reply:
Went with Email instead of ID as discussed